website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: tsp-client usage
+title: Supported commands
```

Comment:
```suggestion
title: Commands
```

Reply:
actually thats not correct, it does more than show the commands, 
<img width="289" alt="image" src="https://github.com/user-attachments/assets/6e711d79-65ec-461d-9a45-a2874a63addf">